packages/grid/x-data-grid/src/models/params/gridCellParams.ts

```diff
@@ -18,11 +18,11 @@ export interface GridCellParams<V = any, R extends GridValidRowModel = any, F =
   /**
    * The cell value, but if the column has valueGetter, use getValue.
    */
-  value: V | undefined;
+  value?: V | undefined;
   /**
    * The cell value formatted with the column valueFormatter.
    */
-  formattedValue: F;
+  formattedValue?: F | undefined;
```

Comment:
Is `| undefined` necessary here since those fields are optional?

Reply:
No
I was to be extra-explicit, but I don't think it has any concrete impact